src/lib/CheckoutServerApi.ts

```diff
@@ -112,6 +112,12 @@ export default class CheckoutServerApi {
         }
 
         if (!fetchResponse.ok) {
+            if (process.env.NODE_ENV === 'development'
```

Comment:
I don't think this place is a good way to put the check, as with this in development the dev will not notice a failing callback url, making testing of error codes or even the callback at all impossible.

I'd say it should only fallback to dummy data if explicitly requested, e.g. by using a constant `callbackUrl` in the request to trigger dummy data.

Reply:
Who do you mean by dev?
For a store dev using a pre-built hub client this line will result in false.

Reply:
Hm, yes you are right. For deployed versions this is `false`. Ok, I guess it's fine then.

Have you tested this? Because I believe in the case where the provided `callbackUrl` is not valid, or rather does not respond at all, `fetch` throws an error both times, which then means that `fetchResponse` will be `undefined` and you'll get an error of  `'Cannot read property ok of undefined'` in the if-condition.

Reply:
I misunderstood. When the second fetch throws, execution stops and we won't even get to line 114. :facepalm: